.kokoro/release.sh

```diff
@@ -60,10 +60,10 @@ fi
 # Push documentation, if we've got the right credentials and haven't been asked to skip
 if [[ -f $DOCS_CREDENTIALS && $SKIP_GOOGLEAPISDEV_UPLOAD = "" ]]
 then
-  echo "Uploading documentation to googleapis.dev"
-  ./uploaddocs.sh releasebuild/nuget releasebuild/docs/output $DOCS_CREDENTIALS docs-staging
+  echo "Uploading documentation to googleapis.dev and devsite"
+  ./uploaddocs.sh releasebuild/nuget releasebuild/docs/output $DOCS_CREDENTIALS docs-staging docs-staging-v2
```

Comment:
```suggestion
  ./uploaddocs.sh releasebuild/nuget releasebuild/docs/output $DOCS_CREDENTIALS docs-staging docs-staging-v2-staging
```

Want to keep the prod bucket for the final version of docs (couldn't think of a better name).

Reply:
Whoops - got confused with the double staging.